hermes/probe/delete/delete.go

```diff
@@ -55,7 +59,7 @@ func DeleteFile(ctx context.Context, fileID int32, target *probe.Target, client
 	bucket := target.Target.GetBucketName()
 
 	if fileID <= 10 || fileID >= 51 {
-		return fileID, fmt.Errorf("delete(%q, %q) failed; status invalidArgument: expected fileID %d to be within valid inclusive range: 11-50", bucket, fileID, fileID)
+		return fileID, fmt.Errorf("delete(%q, %q) failed; invalidArgument: expected fileID %d to be within valid inclusive range: 11-50", bucket, fileID, fileID)
```

Comment:
Nit: Replace "invalidArgument" with "invalid argument". "invalidArgument" suggests that it's some kind of symbol, which it isn't.

Reply:
Okay, changed.